src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1497,6 +1500,11 @@ public async Task RotateApiKeyAsync(Organization organization)
             await ReplaceAndUpdateCache(organization);
         }
 
+        public async Task DeleteSsoUserAsync(SsoUser ssoUser) {
+            await _eventService.LogSsoUserEventAsync(ssoUser, EventType.SsoUser_Deleted);
```

Comment:
Logging the event comes _after_ the event occurs (the event may fail to complete/execute, etc.)

Reply:
fixed